pytorch_lightning/trainer/training_loop.py

```diff
@@ -269,6 +269,14 @@ def _check_training_step_output(self, training_step_output):
             if training_step_output.grad_fn is None:
                 # TODO: Find why - RuntimeError: Expected to mark a variable ready only once ...
                 raise MisconfigurationException("In manual optimization, `training_step` should not return a Tensor")
+            elif self.trainer.automatic_optimization:
```

Comment:
Isn't this one indentation level too deep?

Reply:
Whoops, you're right